0-setup-libraries.ipynb

```diff
@@ -7,16 +7,15 @@
     "Setup Instructions\n",
     "==========\n",
     "\n",
-    "These instructions need to be run once to load the requisite libraries for the tutorial.\n",
+    "These instructions install `miniconda` and the requisite libraries for the tutorial in a `conda` environment, which are indepenent python installation.\n",
```

Comment:
"which are independent" --> "which is an independent..."

Reply:
Fixed.